packages/api-explorer/src/ResponseSchemaBody.jsx

```diff
@@ -51,24 +51,23 @@ class ResponseSchemaBody extends React.Component {
         <tr key={Math.random().toString(10)}>
           <th
             style={{
-              whiteSpace: 'nowrap',
-              width: '30%',
               paddingRight: '5px',
               textAlign: 'right',
-              overflow: 'hidden',
+              overflow: 'visible',
+              wordBreak: 'break-all',
```

Comment:
`break-word` will prefer to keep words intact while allowing breaks.

break-all: pricingOptions = pricingOption (break) s
![Screen Shot 2020-11-24 at 12 06 08 PM](https://user-images.githubusercontent.com/5341618/100145867-d670ac80-2e4d-11eb-8846-1adda2e82085.png)

break-word: pricingOptions kept intact
![Screen Shot 2020-11-24 at 12 06 13 PM](https://user-images.githubusercontent.com/5341618/100145910-e6888c00-2e4d-11eb-9fc0-b4e7eede2d68.png)

Doesn't work in IE, but we could have both and IE will take the break-all fallback.

```
wordBreak: 'break-all',
wordBreak: 'break-word',
```

Reply:
k i'll use both

Reply:
@runnabro Updated!